tensorflow/lite/delegates/BUILD

```diff
@@ -20,6 +20,16 @@ package(
     licenses = ["notice"],  # Apache 2.0
 )
 
+config_setting(
```

Comment:
nit: Do we want to add a comment for each of these `config_setting`s to indicate what they enable and when they should be used, i.e. for building `tensorflow_lite_c` (cc_library) and `TensorFlowLite` (swift_library)?

Reply:
So now that these actually aren't used in the iOS BUILD file, I've moved them directly into the Swift BUILD file, lmk what you think

Reply:
That's a good point! I think the plan may be to add delegation support to `//tensorflow/lite/experimental/objc:TensorFlowLite` in the future, in which case it would probably be better to move the configs to `//tensorflow/lite/experimental/ios/BUILD` so it can be shared between Swift and ObjC. 

@teijeong / @yyoon to confirm.

Reply:
Happy to move it now, or we can as needed